nifi-nar-bundles/nifi-amqp-bundle/nifi-amqp-processors/src/main/java/org/apache/nifi/amqp/processors/ConsumeAMQP.java

```diff
@@ -112,6 +115,17 @@ public class ConsumeAMQP extends AbstractAMQPProcessor<AMQPConsumer> {
         .defaultValue("10")
         .required(true)
         .build();
+    static final PropertyDescriptor PREFETCH_COUNT = new PropertyDescriptor.Builder()
+        .name("prefetch.count")
+        .displayName("Prefetch Count")
+        .description("The maximum number of unacknowledged messages for the consumer. If consumer has this number of unacknowledged messages, AMQP broker will "
+               + "no longer send new messages until consumer acknowledges some of the messages already delivered to it."
+               + "Allowed values: from 0 to 65535. 0 means no limit")
+        .addValidator(StandardValidators.NON_NEGATIVE_INTEGER_VALIDATOR)
```

Comment:
@MooseTheBrown Thanks for adding `Prefetch Count` property!
Please note, there is a standard validator for checking if a number is within a range: https://github.com/apache/nifi/blob/5b664147eeccd9d2a8ded9040a3d64c96e70d639/nifi-commons/nifi-utils/src/main/java/org/apache/nifi/processor/util/StandardValidators.java#L758

Reply:
Corrected. Used standard long validator instead of customValidate in ConsumeAMQP

Reply:
Thanks @MooseTheBrown! Could you please also remove the now unused imports? The build fails with Checkstyle violations. You can check it with running `mvn clean verify -P contrib-check` in `nifi-amqp-bundle` directory.

Reply:
Yes, thanks for pointing this out. Removed unused imports.